### PR TITLE
remove pytz dependency (due to packaging issues)

### PR DESCRIPTION
**Pull request by stdweird**

### CODE REVIEW

**Comment by stdweird** [comment]
@wpoely86 can you test this on el7?

File: lib/vsc/administration/ldapsync.py [hunk 1]
```diff
@@ -21,7 +21,13 @@
 
 from vsc.utils.py2vs3 import HTTPError
 
-import pytz as timezone
+try:
+    from datetime import timezone
+    tz_utc = timezone.utc
+else:
```
Comment (wpoely86) [inline lib/vsc/administration/ldapsync.py:27]: this isn't valid python syntax. Use except?

**Comment by wpoely86** [comment]
will test, please give it a version bump too.

**Comment by stdweird** [comment]
@wpoely86 heh, not sure what happened with that commit. anyway, fixed

**Comment by wpoely86** [comment]
@stdweird For some reason your fork is vsc-administration-1?

The deps are gone (all of them):

```
[ward@build7 ~]$ rpm --requires -qp vsc-administration-1/dist/python-vsc-administration-1-2.4.2-1.noarch.rpm 
/usr/bin/python-stripped-env
python(abi) = 2.7
rpmlib(PartialHardlinkSets) <= 4.0.4-1
rpmlib(FileDigests) <= 4.6.0-1
rpmlib(PayloadFilesHavePrefix) <= 4.0-1
rpmlib(CompressedFileNames) <= 3.0.4-1
rpmlib(PayloadIsXz) <= 5.2-1
```

**Comment by stdweird** [comment]
@wpoely86 i was asking to test the patch in the ldapsync. i know it removes the deps, but i can't  test the actual patch

**Comment by wpoely86** [comment]
OK, I didn't get that.

It works fine here.

**Comment by wpoely86** [comment]
Euh, what's the plan here? Now the dependencies are gone?

**Comment by stdweird** [comment]
@wpoely86 to fix them again. i see that i only removed the comment on the makesetupcfg instead of the whole line. `enum34` had no rpm dep, not sure why it was used; but the lockfile will probably need a fix again to generate the proper rpm name. ah, what a mess this is...

**Pull request merged**